src/main/java/mate/academy/AuthenticationService.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy;
+
+import mate.academy.model.User;
+
+public class AuthenticationService {
+    private final UserService userService = new UserService();
+
+    public boolean login(String email, String password) {
+        User user = userService.findByEmail(email);
+
+        return user != null && !user.getPassword().equals(password);
```

Comment:
why passwords should not match?
```suggestion
        return user != null && user.getPassword().equals(password);
```